drop/template/Scripts/rnaVariantCalling/Overview.R

```diff
@@ -0,0 +1,44 @@
+#'---
+#' title: RNA Variant Calling
+#' author: Nick Smith
+#' wb:
+#'  log:
+#'    - snakemake: '`sm str(tmp_dir / "RVC" / "Overview.Rds")`'
+#'  input:
+#'    - vcfFilesBatch: '`sm expand(cfg.getProcessedDataDir() +
+#'                      "/rnaVariantCalling/out/all_samples_haplocaller/" + 
+#'                      "{batchID}_all_samples.genotyped.filtered_clean.vcf.gz",
+#'                  batchID=cfg.RVC.groups)`'
+#'    - countFiles: '`sm expand(cfg.getProcessedDataDir() +
+#'                      "/rnaVariantCalling/out/sample_haplocaller/" + 
+#'                      "{sample}/{sample}_variant_counts.txt",
+#'                  sample=cfg.RVC.batchIDs,min_alt=getMinAlt())`'
+#'    - vcfFilesMasked: '`sm expand(cfg.getProcessedDataDir() +
+#'                      "/rnaVariantCalling/out/sample_haplocaller/" + 
+#'                      "{sample}/{sample}.genotyped.filtered.basic{min_alt}.masked.vcf.gz",
+#'                  sample=cfg.RVC.batchIDs,min_alt=getMinAlt())`'
+#' output:
+#'   html_document:
+#'    code_folding: hide
+#'    code_download: TRUE
+#'---
+
+#+ echo=FALSE
+library(data.table)
+library(ggplot2)
+
+saveRDS(snakemake, snakemake@log$snakemake)
+
+#+ eval=TRUE, echo=FALSE
```

Comment:
I think `sep` is not needed. You could simplify lines 33-39 into the following:
`plot_table <- lapply(snakemake@input$countFiles, fread) %>% rbindlist()`